### PR TITLE
Added DND test suit

### PR DESCRIPTION
**Pull request by selenai**
### Issue number

FRON-321

### CODE REVIEW

File: e2e/calling.out.spec.ts [hunk 1]
```diff
@@ -13,11 +13,11 @@ test.describe('Calling out', () => {
     helpersA = new HelpFunctions(pageUserA);
     helpersB = new HelpFunctions(pageUserB);
 
-    await pageUserA.goto(`${process.env.DEMO_URL}`);
+    await pageUserA.goto(`${process.env.DEMO_URL}`, { timeout: 0 });
```
Comment (Dagg-h) [inline e2e/calling.out.spec.ts:16]: Does this really help? Otherwise the timeout can be removed :)

File: e2e/dnd.spec.ts [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    await helpersA.registerUser(`${process.env.USER_A}`, `${process.env.PASSWORD_A}`);
+    await helpersA.assertAccountStatus('connected');
+
+    await pageUserB.goto(`${process.env.DEMO_URL}`, { timeout: 0 });
+    await helpersB.registerUser(`${process.env.USER_B}`, `${process.env.PASSWORD_B}`);
+    await helpersB.assertAccountStatus('connected');
+
+    await pageUserC.goto(`${process.env.DEMO_URL}`, { timeout: 0 });
+    await helpersC.registerUser(`${process.env.USER_C}`, `${process.env.PASSWORD_C}`);
+    //await helpersC.assertAccountStatus('connected');
```
Comment (Dagg-h) [inline e2e/dnd.spec.ts:29]:
```suggestion
    await helpersC.assertAccountStatus('connected');
```

File: e2e/helpers/helpers.ts [hunk 3]
```diff
@@ -47,6 +48,7 @@ export class HelpFunctions {
     this.dialerInput = page.locator('c-dialer [data-selector="input"]');
     this.subscriptionInput = page.locator('c-contacts [data-selector="input"]');
     this.accountStatus = page.locator('[data-selector="clientStatus"]');
+    this.dndToggle = page.locator('.switch');
```
Comment (Dagg-h) [inline e2e/helpers/helpers.ts:51]:
```suggestion
    this.dndToggle = page.locator('[data-action="dndToggle"]');
```

Comment (Dagg-h) [reply e2e/helpers/helpers.ts:51]: Check if it works though :sweat_smile:

File: e2e/helpers/helpers.ts [hunk 4]
```diff
@@ -115,6 +117,8 @@ export class HelpFunctions {
   }
 
   async callNumber(number: string) {
+    await this.dialerInput.click({ clickCount: 3 });
```
Comment (Dagg-h) [inline e2e/helpers/helpers.ts:120]: Without these two lines it used to work fine right?

File: e2e/helpers/helpers.ts (hunk 3 above)
Comment (selenai) [reply e2e/helpers/helpers.ts:51]: I tried this locator, but it gives an error:

waiting for selector "[data-action="dndToggle"]"
  selector resolved to hidden <input type="checkbox" data-action="dndToggle"/>
attempting click action
  waiting for element to be visible, enabled and stable

**Pull request merged**